hr_contract_amendment/models/hr_contract.py

```diff
@@ -0,0 +1,92 @@
+# -*- coding: utf-8 -*-
+#   Copyright (C) 2018 EBII (https://www.saaslys.com).
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from odoo import fields, models, _
+
+
+class HrAmendmentContractChanges(models.Model):
```

Comment:
use the same name as the model HRContractAmendmentChanges

Reply:
name of file ? 
make two hr_contract.py & hr_amendment.py  ?